commercetools-convenience/src/test/java/io/sphere/sdk/meta/CategoryDocumentationIntegrationTest.java

```diff
@@ -74,31 +74,105 @@ public static void beforeClass() throws IOException {
     @Test
     public void fetchAll() throws Exception {
         final CompletionStage<List<Category>> categoriesStage = QueryExecutionUtils.queryAll(client(), CategoryQuery.of(), 500);
-        final List<Category> categories = SphereClientUtils.blockingWait(categoriesStage, Duration.ofMinutes(5));
+        final List<Category> categories = SphereClientUtils.blockingWait(categoriesStage, Duration.ofMinutes(1));
         assertThat(categories)
                 .hasSize(15)
                 .matches(cats -> cats.parallelStream().anyMatch(cat -> cat.getSlug().get(ENGLISH).equals("boots-women")));
     }
+
+    @Test
```

Comment:
I like your tests 👍

Reply:
the tests are mostly done by @heshamMassoud though :)

I just re-factored the tests after re-factored the solution :)